### DISCUSSION TITLE
Install has wrong

### DISCUSSION THREAD

**Discussion by WZF-918** [biostars/biostar-handbook-issues#276] [Q&A]
#
# 7/10 Installing bioinformatics software
#
  Traceback (most recent call last):
  File "/root/miniconda3/condabin/mamba", line 7, in <module>
    from mamba.mamba import main
  File "/root/miniconda3/lib/python3.11/site-packages/mamba/mamba.py", line 49, in <module>
    import libmambapy as api
  File "/root/miniconda3/lib/python3.11/site-packages/libmambapy/__init__.py", line 7, in <module>
    raise e
  File "/root/miniconda3/lib/python3.11/site-packages/libmambapy/__init__.py", line 4, in <module>
    from libmambapy.bindings import *  # noqa: F401,F403
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ImportError: libarchive.so.13: cannot open shared object file: No such file or directory
I have tried several times to close the terminal and re-download, but the above problems still occur, I really do not know how to deal with, please help

**Comment by ialbert** [comment]
Your error shows the path of `/root/miniconda3`

why is it installed into a path that starts with the word  `root`? That sounds like the instructions were not followed the same way as in the book.

What operating system are you using?

**Reply by WZF-918** [reply]
The operating system I use is Ubuntu, I seem to have problems with the installation of Ubuntu but I can't find the problem

**Reply by ialbert** [reply]
don't run the commands as root, that can completely mess up the entire system

create a local user and use that local user account

**Reply by WZF-918** [reply]
I have tried it.,but it's showing" miniconda-installer.sh: Permission denied"

**Reply by ialbert** [reply]
That means you have downloaded as root and trying to run as a different user.

Make a new user account, and start from scratch. In that case it will work - the installation instructions are tested each year with thousands of new users. They do work - but the instructions must be followed very closely in case you have no prior experience with Unix.

**Reply by WZF-918** [reply]
If I uninstall Ubuntu and reinstall it, can I create a local user to complete the software download?

**Reply by ialbert** [reply]
that would be a good way to go about it.
